### PR TITLE
Mention "arithmetic"

### PR DESCRIPTION
**Pull request by kitzberger**
Resolves: #67

### CODE REVIEW

File: docs/guide/usage/editing.md [hunk 1]
```diff
@@ -78,6 +78,28 @@ name a few:
...
+
+Best used when having multiple selections.
+
+| Expression         | Selection   | Result                         |
+| ------------------ | ----------- | ------------------------------ |
+| `1+i`              |             | 1, 2, 3, 4, 5, 6, 7, ...       |
+| `30+(i%3)`         |             | 30, 31, 32, 30, 31, 32, ...    |
+| `(x*2)+i`          | 10, 20, 30  | 20, 41, 62                     |
+| `math.ceil(x)`     | 0.25, 2.25  | 1, 3                           |
+| `len(z)`           | house, tree | 5, 4                           |
```
Comment (michaelblyons) [inline docs/guide/usage/editing.md:93]: `z` or `s` here and below?

Comment (kitzberger) [reply docs/guide/usage/editing.md]: Ah, damn, yes, you're right. Thanks ;)

**Review by FichteFoll — APPROVED** [review]
Thanks, this is nice and short.

**Comment by kitzberger** [comment]
> should've linked the py3.8 docs, though

@jrappen, you're probably refering to this here, right? https://www.sublimetext.com/docs/api_environments.html#python-version

**Pull request merged**